.env.sample

```diff
@@ -4,7 +4,7 @@ FILE_OUT_PATH=./out
 DUNE_API_KEY=
 
 # Safe Transaction Service Requirements.
-SAFE_ADDRESS=0xA03be496e67Ec29bC62F01a428683D7F9c204930
+PAYOUTS_SAFE_ADDRESS=0xA03be496e67Ec29bC62F01a428683D7F9c204930
 NETWORK=mainnet
```

Comment:
Could you add a comment with the options here?
Is it {mainnet, xdai, arbitrum_one} or {ethereum, gnosis, arbitrum}?

Reply:
I added a comment.